### DISCUSSION TITLE
Collaboration ideas

### DISCUSSION THREAD

**Discussion by AbdelStark** [keep-starknet-strange/madara#899] [Ideas]
# Ideas for collaboration

- Madara - #247 
- #896
- #897
- #898
- #900 
- #1147 
- Starknet fee market experiments and analysis: `TODO - create detailed issue`
- Efficient Rust libraries to compute Starknet state commitment: `TODO - created detailed issue`, relates to #900 (this would be the primitives used by it in the client, those primitives can also be used by the committer)
- Integrate [stone](https://github.com/starkware-libs/stone-prover) prover in Madara: `TODO - created detailed issue`
- Integrate [sandstorm](https://github.com/andrewmilson/sandstorm) prover in Madara: `TODO - created detailed issue`

**Comment by jommi9** [comment]
Hey, what is this about? We've been working with Tendermint, Substrate as well as Barhwal and Bullshark at our Consensus team at https://equilibrium.co/ for a while now. Is there a chance to contribute to the above discussions and work somehow?

**Reply by AbdelStark** [reply]
of course, with pleasure. reach out to me on telegram
